streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStore.java

```diff
@@ -87,11 +115,13 @@ public boolean persistent() {
 
     @Override
     public V get(K key) {
+        maybeInitialize();
```

Comment:
Instead of doing this, you could have an accessor for `inner` that forced initialisation.